docs/contributing.rst

```diff
@@ -261,6 +261,8 @@ All integrations live in ``ddtrace/contrib/`` and contain at least two files,
 
 It is preferred to keep as much code as possible in ``patch.py``.
 
+All spans generated by the integration must add the tag ``component:<integration_name>`` to each span.
+`Example of component tag being set in Flask integration <https://github.com/DataDog/dd-trace-py/blob/1.x/ddtrace/contrib/flask/patch.py#L486-L487>`_.
```

Comment:
If the 1.x branch ever changes, this link could end up pointing to something other than the intended bit of code. You can turn it into a GitHub permalink by pressing `y` on that page.

Reply:
Today I learned 😮

Reply:
same, merged Yun's suggestion, should be good for final approvals